src/NuGetPackageVerifier/Program.cs

```diff
@@ -25,6 +25,13 @@ public static int Main(string[] args)
 
             // TODO: Get this from the command line
             var ignoreAssistanceMode = IgnoreAssistanceMode.ShowAll;
+            var hideInfoLogs = false;
+
+            if (args.Length > 0 && string.Equals("--quiet", args[0], StringComparison.OrdinalIgnoreCase))
```

Comment:
Not super vested in this, but do we want to make the default `Normal` and hide the random spew we get today `Info`?

```
N: Read 2 package set(s) from C:\BuildAgent\work\172afee273357bca\NuGetPackageVerifier.json 
N: Found 1 packages in C:\BuildAgent\work\172afee273357bca\artifacts\build 
I: Processing package set 'adx' with 1 package(s)
I: Analyzing Microsoft.AspNetCore.Antiforgery (1.2.0-t003a5150a)
I: No issues found with package Microsoft.AspNetCore.Antiforgery (1.2.0-t003a5150a)

I: Processing package set 'Default' with 0 package(s)
I: Showing JSON for ignore content: < -- Info when no error, else, normal
I: {}

N: SUMMARY: 0 error(s) and 0 warning(s) found
N: Total took 568ms
```